collections/sort_by.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright 2018-2021 the Deno authors. All rights reserved. MIT license.
+
+/**
+ * Returns all elements in the given collection, sorted by their result using the given selector
+ *
+ * Example:
+ *
+ * ```typescript
```

Comment:
We prefer `ts` tag for typescript example block. ref: https://github.com/denoland/deno_std/pull/1050#discussion_r673884120

Reply:
Fixed